### DISCUSSION TITLE
Top letter row with numbers

### DISCUSSION THREAD

**Discussion by Dimas-sc** [AnySoftKeyboard/AnySoftKeyboard#2867] [Q&A]
Hi,
If you maintain pressed the letters on the top row of the keyboard you can use it to input the numbers, but is it possible to view which number before tapping on it?
Like in other keyboards:
![imatge](https://user-images.githubusercontent.com/2415679/119105722-ea245680-ba1d-11eb-9f11-61d624e9c760.png)

**Comment by douellette20** [comment]
I believe this is found in the **UI** -> **Pick and configure keyboard themes** area of the settings; several themes have the long-press characters in smaller background text.
